pkg/cmd/roachtest/tests/kv.go

```diff
@@ -190,7 +197,15 @@ func registerKV(r registry.Registry) {
 		// CPU overload test, to stress admission control.
 		{nodes: 1, cpus: 8, readPercent: 50, concMultiplier: 8192},
 		// IO write overload test, to stress admission control.
-		{nodes: 1, cpus: 8, readPercent: 0, concMultiplier: 4096, blockSize: 1 << 16 /* 64 KB */},
+		//
+		// jemallocReleaseFaster is set to true due to OOMs observed in
+		// https://github.com/cockroachdb/cockroach/issues/125769. There is
+		// unavoidable 20% internal fragmentation in the Pebble block cache, since
+		// a 64KB value size causes a 64+KB sstable block, which allocates
+		// from the 80KB size class in jemalloc. So the allocated bytes from jemalloc
+		// by the block cache are 20% higher than configured. By setting this flag to true,
+		// we reduce the (resident-allocated) size in jemalloc.
+		{nodes: 1, cpus: 8, readPercent: 0, concMultiplier: 4096, blockSize: 1 << 16 /* 64 KB */, jemallocReleaseFaster: true},
```

Comment:
Do we expect the throughput to drop? Either way, a roachperf annotation would probably be insightful.